contracts/src/CrossChainApplications/examples/ExampleMessenger/ExampleCrossChainMessenger.sol

```diff
@@ -54,9 +54,10 @@ contract ExampleCrossChainMessenger is ReentrancyGuard, TeleporterOwnerUpgradeab
         bytes32 indexed sourceBlockchainID, address indexed originSenderAddress, string message
     );
 
-    constructor(address teleporterRegistryAddress)
-        TeleporterOwnerUpgradeable(teleporterRegistryAddress)
-    {}
+    constructor(
+        address teleporterRegistryAddress,
+        address initialOwner
```

Comment:
To reduce the diff and given that these are all only example applications, I have a slight preference for using `msg.sender` rather than adding an extra constructor parameter to all of the examples. I think it raises more questions than benefits in the examples.

"Owner" is also a scary term in smart contracts since it applies they have significant admin-type access. Is there a better term we could use to make clear the owner only has the ability to set the minimum Teleporter version allowed to be used?

Reply:
Maybe something like `teleporterUpgradeableAdmin`?

Reply:
My thinking is that the example cross chain apps should illustrate a standard for how apps should integrate with Teleporter, and that it'd be a good standard for apps to have an `initialOwner` argument. 

No strong pref for naming, good with `teleporterUpgradeableAdmin`, `teleporterManager`, `teleporterUpdater`, etc.

Reply:
`teleporterManager` SGTM 👍